habootstrap-formula.spec

```
@@ -32,6 +32,7 @@ Source0:        %{name}-%{version}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 Requires:       salt-saphana
+Requires:       salt-minion >= 2018.3.0
```

@arbulu89 I see a "chicken/egg" problem here... When the formula is installed on the salt-master, AFAIR there is no salt-minion installed there... So the installation would fail. We need to check if the salt packages provides some virtual "provides", like "salt" only that is common to all salt packages, that we could use in this case...

---

Good point! I didn't consider this point. I will see how we can fix that use case

---

Ok, checking the salt packaging, master and minion depend on a salt package. So, we can include the same dependency here and it will be independent of where the formula is being installed.

```suggestion
Requires(pre):       salt >= 2018.3.0
```
The `(pre)` clause is to enforce that salt package is installed before the formula package.

---

@diegoakechi Could we use Suggests instead of Requires? As If you want to use the formula in salt master, you shouldn't be forced to install the salt-minion in this machine.

---

Ok, I found something better:

`Requires: (pkgA or pkgB or pkgC)`

https://rpm.org/user_doc/boolean_dependencies.html